sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -439,3 +439,117 @@ ALTER TABLE `onsite_mail` CHANGE `owner` `owner` varchar(128) DEFAULT NULL;
+(33, 'POSTCARD', 'SCHEDULED', 'Postcard Campaign Event is scheduled.', '<span class="btn scheduled" title="Postcard Campaign Event is scheduled."><i class="fa fa-image fa-fw"></i></span>', ''),
+(36, 'AVM', 'READ', 'AVM delivered', '<span class="btn" title="AVM completed - waiting for manual response" aria-label="AVM Delivered" style="padding:5px;background-color:#146abd;"><i class="fa fa-phone fa-fw" aria-hidden="true"></i></span>', ''),
+(37, 'SMS', 'CALLED', 'Callback Completed', '<span class="btn btn-success" style="padding:5px;background-color:#146abd;" title="Patient requests Office Call: COMPLETED">\r\n<i class="fa fa-flag fa-fw"></i></span>\r\n', '');
+
+
+#IfNotTable medex_outgoing
+CREATE TABLE IF NOT EXISTS `medex_outgoing` (
+  `msg_uid` int(11) NOT NULL AUTO_INCREMENT,
+  `msg_pc_eid` varchar(11) NOT NULL,
+  `campaign_uid` int(11) NOT NULL DEFAULT '0',
+  `msg_date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `msg_type` varchar(50) NOT NULL,
+  `msg_reply` varchar(50) DEFAULT NULL,
+  `msg_extra_text` text,
+  PRIMARY KEY (`msg_uid`),
+  UNIQUE KEY `msg_eid` (`msg_uid`,`msg_pc_eid`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 AUTO_INCREMENT=1 ;
+#Endif
+
+#IfNotTable medex_prefs
+CREATE TABLE IF NOT EXISTS `medex_prefs` (
+  `MedEx_id` int(11) DEFAULT '0',
+  `ME_username` varchar(100) DEFAULT NULL,
+  `ME_api_key` text,
+  `ME_facilities` varchar(50) DEFAULT NULL,
+  `ME_providers` varchar(100) DEFAULT NULL,
+  `ME_hipaa_default_override` varchar(3) DEFAULT NULL,
+  `PHONE_country_code` int(4) NOT NULL DEFAULT '1',
+  `MSGS_default_yes` varchar(3) DEFAULT NULL,
+  `POSTCARDS_local` varchar(3) DEFAULT NULL,
+  `POSTCARDS_remote` varchar(3) DEFAULT NULL,
+  `LABELS_local` varchar(3) DEFAULT NULL,
+  `LABELS_choice` varchar(50) DEFAULT NULL,
+  `combine_time` tinyint(4) DEFAULT NULL,
+  `notify` varchar(100) DEFAULT NULL,
+  `MedEx_lastupdated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  UNIQUE KEY `ME_username` (`ME_username`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+#Endif
+
+#IfNotTable medex_recalls
+CREATE TABLE IF NOT EXISTS `medex_recalls` (
+  `r_ID` int(11) NOT NULL AUTO_INCREMENT,
+  `r_PRACTID` int(11) NOT NULL,
+  `r_pid` int(11) NOT NULL COMMENT 'PatientID from pat_data',
+  `r_eventDate` date NOT NULL COMMENT 'Date of Appt or Recall',
+  `r_facility` int(11) NOT NULL,
+  `r_provider` int(11) NOT NULL,
+  `r_reason` varchar(255) DEFAULT NULL,
+  `r_created` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00' ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`r_ID`),
+  UNIQUE KEY `r_PRACTID` (`r_PRACTID`,`r_pid`)
+) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=168 ;
+#Endif
+
+#IfNotRow background_services name MedEx
+INSERT INTO `background_services` (`name`, `title`, `active`, `running`, `next_run`, `execute_interval`, `function`, `require_once`, `sort_order`) VALUES
+('MedEx', 'MedEx Messaging Service', 29, 0, '2017-05-09 17:39:10', 60, 'start_MedEx', '/library/MedEx/medex_background.php', 100);
```

Comment:
btw, as an example showing the site from multisite is set as the first parameter:
execute_background_services.php default